extensions/cache/deployment/src/test/java/io/quarkus/cache/test/deployment/CacheConfigTest.java

```diff
@@ -38,6 +43,15 @@ public void testConfig() {
         assertTrue(cache.getCacheInfo().metricsEnabled);
     }
 
+    @Test
+    @DisplayName("Check that the io.quarkus.cache.CaffeineCache#put(Object, CompletableFuture) works as expected.")
+    public void putShouldPopulateCache() {
```

Comment:
`CacheConfigTest` is focused on testing the build time config. Could you please move that test to `io.quarkus.cache.test.runtime.ProgrammaticApiTest`?

Reply:
Good catch, done!